Bottom: d9da4d4dcd9730b4dac15902d00218f841b21e07
Top:    e303f20481c9085e70236c012bbfa6886fd92bf1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 01:29:20 +0100

Refresh of expose-draw-c-internal-drawing

---

diff --git a/src/draw.c b/src/draw.c
index 81bf5ec..1a7171e 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -92,10 +92,7 @@ static int DrawLayerGroup (int, const BoxType *);
 static void AddPart (void *);
 static void SetPVColor (PinTypePtr, int);
 static void DrawEMark (ElementTypePtr, LocationType, LocationType, bool);
-static void DrawMask (int side, BoxType *);
-static void DrawPaste (int side, BoxType *);
 static void DrawRats (BoxType *);
-static void DrawSilk (int side, const BoxType *);
 
 /*--------------------------------------------------------------------------------------
  * setup color for pin or via
